src/Mod/PartDesign/Gui/CommandBody.cpp

```diff
@@ -189,41 +189,45 @@ void CmdPartDesignBody::activated(int iMsg)
     openCommand(QT_TRANSLATE_NOOP("Command", "Add a Body"));
 
     std::string bodyName = getUniqueObjectName("Body");
+    const char* bodyString = bodyName.c_str();
 
     // add the Body feature itself, and make it active
-    doCommand(Doc,"App.activeDocument().addObject('PartDesign::Body','%s')", bodyName.c_str());
+    doCommand(Doc,"App.activeDocument().addObject('PartDesign::Body','%s')", bodyString);
+    // set Label for i18n/L10N
+    QByteArray labelByteArray = QObject::tr("Body").toUtf8();
```

Comment:
You should be able to use just `tr()` here, right? Rather than stripping off the context with `QObject`?

Reply:
Unfortunately not. If I do so I get the following error message from the compiler:
`error: ‘tr’ was not declared in this scope; did you mean ‘tm’?`

Reply:
OK, I thought this was a QObject, but it must not be.

Reply:
Hmm, I'm pretty new to C++ and the whole FreeCAD code base, but wouldn't this exactly be the use case for the Q_DECLARE_TR_FUNCTIONS macro? We could declare the context in this macro and use `tr()` instead of `QObject::tr()`.

Reply:
OK, I will try that here. How is the workflow now? Just adding another commit to this PR or creating a new PR?

Reply:
Now that this is merged, you'll have to make a new PR for anything further you want to work on.

Reply:
OK, will do so.